### DISCUSSION TITLE
Trouble with Football Player Tracking for my own video (non-kaggle)

### DISCUSSION THREAD

**Discussion by mihirmath** [roboflow/notebooks#134] [Q&A]
Hello,

I am trying to follow the tutorial for this video https://www.youtube.com/watch?v=QCG8QMhga9k. I got everything to work for the video downloaded from kaggle.

I'm now having difficulty running the code on my own mp4 file. The only things I changed were the file name of the video clip and the number of frames to loop over. All is good until after the custom model video with the bounding boxes (this works with my video). However, when I try to run the code after that when I try to do plot_image() it does not display anything but the cell runs. Then, when I proceed as if it worked, the final video is not able to export properly since I think the video does not exist.

Would appreciate any help or if anyone has code I could look at for their own non-kaggle video would be helpful!

**Comment by mihirmath** [comment]
@SkalskiP

**Comment by SkalskiP** [comment]
Hi, @mihirmath 👋🏻! Sorry for the late response. I didn't notice your message. What is your video resolution?

**Reply by mihirmath** [reply]
@SkalskiP Thanks so much for getting back to me! My resolution is as follows:
<img width="228" alt="Screenshot 2023-05-26 at 9 09 14 AM" src="https://github.com/roboflow/notebooks/assets/64558607/3fb1998f-3fa4-4544-96b1-dc1dfd357a08">
It appears to be different than the sample video so that could very well be the problem. 
<img width="1726" alt="Screenshot 2023-05-26 at 9 10 31 AM" src="https://github.com/roboflow/notebooks/assets/64558607/565dc4d2-0e23-4113-ae5a-465f312063ae">
I do get the custom dataset to work with the bounding boxes though.

**Reply by mihirmath** [reply]
@SkalskiP I also wanted to find a way to track one player throughout a clip (or a whole game). I notice that the unique index for a player can change over the course of a clip. Is there a way to alter the code to only track a player of the user's choosing? How might I approach this? I was thinking of filtering by the unique id of that player and then following that id but if that id is changing in between clips this may not work.

Best,
Mihir

**Reply by mihirmath** [reply]
@SkalskiP

**Reply by SkalskiP** [reply]
Hi, @mihirmath 👋🏻! Sorry for the late response. Did you change this part of the code:

```python
video_config = VideoConfig(
    fps=30, 
    width=1920, 
    height=1080)
video_writer = get_video_writer(
    target_video_path=TARGET_VIDEO_PATH, 
    video_config=video_config)
```

It looks like you need to adjust for different resolutions. Input and output frames should have the same dimensions.

**Reply by SkalskiP** [reply]
>  also wanted to find a way to track one player throughout a clip (or a whole game). I notice that the unique index for a player can change over the course of a clip. Is there a way to alter the code to only track a player of the user's choosing? How might I approach this? I was thinking of filtering by the unique id of that player and then following that id, but if that id is changing in between clips, this may not work.

With that camera angle, it will be hard to prevent those accidental id swaps. They happen because players occlude each other. The easiest way to prevent that is to put camera as high as possible.
